auth/src/main/java/com/firebase/ui/auth/ResultCodes.java

```diff
@@ -0,0 +1,33 @@
+package com.firebase.ui.auth;
+
+import android.app.Activity;
+
+/**
+ * Result codes returned when using {@link AuthUI.SignInIntentBuilder#build()} with
+ * {@code startActivityForResult}.
+ */
+public final class ResultCodes {
+    private ResultCodes() {
+        // We don't want people to initialize this class
+    }
+
+    /**
+     * Sign in succeeded
+     **/
+    public static final int OK = Activity.RESULT_OK;
```

Comment:
I decided to unify result codes and make `ResultCodes` a central place for those.

I decided to use the existing activity result codes to not break anything.